com.sap.cloud.lm.sl.common/src/main/java/com/sap/cloud/lm/sl/common/model/xml/ListWrapper.java

```diff
@@ -17,7 +17,7 @@
 
     @XmlElements({ @XmlElement(name = "value", type = String.class), @XmlElement(name = "list", type = ListWrapper.class),
         @XmlElement(name = "map", type = MapWrapper.class), })
-    private List<Object> list = new ArrayList<>();
```

Comment:
Are you **absolutely** sure this won't change the XML we produce when serializing lists/maps? If you are, fine, rename it, but also rename the variable on line 28 for consistency.